src/io/opendss.jl

```diff
@@ -1082,6 +1082,16 @@ function _adjust_sourcegen_bounds!(pmd_data)
     pmd_data["gen"]["1"]["pmax"] = _PMs.MultiConductorVector(fill( bound, size(pmd_data["gen"]["1"]["pmin"])))
     pmd_data["gen"]["1"]["qmin"] = _PMs.MultiConductorVector(fill(-bound, size(pmd_data["gen"]["1"]["pmin"])))
     pmd_data["gen"]["1"]["qmax"] = _PMs.MultiConductorVector(fill( bound, size(pmd_data["gen"]["1"]["pmin"])))
+
+    # set current rating of vbranch modelling internal impedance
+    sourcegen = pmd_data["gen"]["1"]
+    sourcebus = pmd_data["bus"][string(sourcegen["gen_bus"])]
+    sid = sourcebus["index"]
+    # the branch modelling the internal impedance, is the only branch connected to the sourcebus
+    vbranch = [br for (id, br) in pmd_data["branch"] if br["f_bus"]==sid || br["t_bus"]==sid][1]
+    vbranch["rate_a"] = _PMs.MultiConductorVector(fill(bound, length(vbranch["rate_a"])))
+    #vbranch["rate_b"] = bound
+    #vbranch["rate_c"] = bound
```

Comment:
you should search for the virtual branch by the name `"sourcebus_vbranch"`, so you won't need L1086-L1090, and you should remove L1093-L1094 since they don't do anything